src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -177,6 +179,21 @@ private[snowflake] class JDBCWrapper {
     DriverManager.getConnection(jdbcURL, jdbcProperties)
   }
 
+  private def setupJDBCOptions(params: MergedParameters, jdbcProperties: Properties): Unit = {
+    if (params.parameters.get("query_result_format").isDefined) {
+      jdbcProperties.put("query_result_format",
+        params.parameters.get("query_result_format").get)
+    }
+
+    if (params.parameters.get("time_output_format").isDefined) {
+      jdbcProperties.put("time_output_format",
+        params.parameters.get("time_output_format").get)
+    }
```

Comment:
I understand your idea, it's doable. But I want to play it simply to keep current change.
Look into the function of getConnector(). For some of the parameters, JDBC parameter names are used directly, but for some of them, separate fields in parameter are used. Even for fields in parameter, both "match" and "if" can be seen.
My understanding is that, if the parameter name in Spark Connector is different to the name in JDBC, a separate field name can be used e.g. "sfrole" -> "role", otherwise, it may not be necessary.
In the future, if we want to enable spark connector user can set any JDBC properties, it may make the parameter class very big.

Reply:
That is legacy issues. At least, don't use String value directly. Try to reference this values from one constant.
We have no plan to directly forward JDBC parameters because JDBC should be transparent in Spark Connector. If you directly forward parameters, customer can crash Spark Connector.